docs/ops/README.md

```diff
@@ -0,0 +1,33 @@
+<!-- 
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# Operation Guide
+This guide covers the info about operate the submarine.
+
+## Submarine Server
+Submarine Server as the core component of submarine, is responsible for manage job life cycle and connect submarine and the resource management system(YARN/Kubernetes). 
+
+For more info see:
+* [Architecture](../design/submarine-server/architecture.md)
+* [Deployment](./server/README.md)
+* [Training](./server/README.md#submit-job)
+* [Q&A](./server/README.md#q&a)
+
+## Submarine SDK
```

Comment:
Suggest to remove this "submarine SDK section". And how about change the directory "ops" to "submarine-server"?